game-service/src/main/java/org/libertybikes/game/core/GameRound.java

```diff
@@ -445,6 +484,15 @@ private void delay(long ms) {
         return clients.entrySet()
                         .stream()
                         .filter(c -> !c.getValue().isPhone)
+                        .filter(c -> !c.getValue().isExternalAi)
+                        .map(s -> s.getKey())
+                        .collect(Collectors.toSet());
+    }
+
+    private Set<Session> getExternalAiSessions() {
```

Comment:
is this new method necessary? Can't we just treat AI clients the same way as non-mobile sessions?